news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Community Mentorship Program Winter 2017 Signups Now Open"
+date: 2017-10-08 12:00:00 +0000
+---
+Have you ever opened up the editor and just looked at everything it offers in confusion? Do you have no idea where to even begin beatmapping? Worry no further, the Community Mentorship Program might be the place for you!
+
+![](/wiki/shared/news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open/banner.png)
+
+Since its inception in July of 2016, the program has gone through five cycles where experienced mappers teach aspiring new mappers everything there is to know about beatmapping, from the fundamentals of handling the editor to abstract and complex concepts of beatmap design, such as structure, flow, and many more!
```

Comment:
L10 can be improved, suggested ``Since its inception during July 2016 [..]``

Reply:
there's a distinct point in time rather than a period when this program came into being so i dont see the need to rephrase this

Reply:
IMO this sound grammatically wrong, might be me though.